example-apps/complex-setup-app/todo-service/service.yml

```diff
@@ -5,7 +5,6 @@ author: test-author
 startup:
   command: echo "does not run"
```

Comment:
Does this command have any use now?

Reply:
It's temporary for now, next PR will remove this